taca/illumina/Runs.py

```diff
@@ -96,7 +94,6 @@ def check_run_status(self):
                 if self.is_unpooled_lane(lane):
                     self._rename_undet(lane, samples_per_lane)
 
-
     def _set_run_type(self):
```

Comment:
I'm guessing these aren't used as well, as they don't seem to be implemented?

Reply:
They're implemented in each subclass instead. Not sure if having them like this in the base class serves a purpose or not, so I thought it would be good to keep them for now.